src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionResponseContent.cs

```diff
@@ -12,26 +12,25 @@ namespace System.Net.Http
     internal sealed class HttpConnectionResponseContent : HttpContent
     {
         private Stream _stream;
-        private bool _consumedStream;
 
         public void SetStream(Stream stream)
         {
             Debug.Assert(stream != null);
             Debug.Assert(stream.CanRead);
-            Debug.Assert(!_consumedStream);
 
             _stream = stream;
         }
 
         private Stream ConsumeStream()
         {
-            if (_consumedStream || _stream == null)
+            Stream stream = _stream;
```

Comment:
You added _consumedStream originally, see: https://github.com/dotnet/corefx/pull/27476

Reply:
I can't argue with myself, can I :)  Wonder why it wasn't failing the tests I said I added.  I'll take another look.